docs/conf/inspircd.conf.example

```diff
@@ -962,10 +963,10 @@
 
 <badnick
          # nick: Nick to disallow. Wildcards are supported.
-         nick="ChanServ"
+         nick="Admin"
```

Comment:
Can you use a nick here that it's likely that people won't use? We've had enough trouble caused by the `root@*` example...